From 18230ff998c5df67117ac99aa6c4a5f7b9a92597 Mon Sep 17 00:00:00 2001
From: Felix Kuehling <Felix.Kuehling@amd.com>
Date: Thu, 30 Mar 2017 13:47:03 -0400
Subject: [PATCH 2010/2094] drm/amdgpu: Don't wait for page table/directory
 validation

Any subsequent page table update will already wait for the fences
on the PT/PD BOs. But it's doing it more efficiently in the GPU
scheduler.

Change-Id: I90cb0fd82a8e0fd16d70250cb0b365c4bb42c389
Signed-off-by: Felix Kuehling <Felix.Kuehling@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index a81fecc..1e533c7 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -433,7 +433,7 @@ static int validate_pt_pd_bos(struct amdgpu_device *adev, struct amdgpu_vm *vm)
 	int ret;
 
 	param.domain = AMDGPU_GEM_DOMAIN_VRAM;
-	param.wait = true;
+	param.wait = false;
 
 	ret = amdgpu_vm_validate_pt_bos(adev, vm, amdgpu_amdkfd_validate,
 					&param);
-- 
2.7.4

